OrbitGl/TimeGraph.cpp

```diff
@@ -704,6 +776,45 @@ void TimeGraph::OnRight() {
   NeedsUpdate();
 }
 
+//----------------------------------------------------------------------------
+void TimeGraph::JumpToBox(const TextBox* a_TextBox) {
+  if (!a_TextBox) return;
+  TextBox* text_box = const_cast<TextBox*>(a_TextBox);
+  Zoom(text_box);
+  Capture::GSelectedTextBox = text_box;
+  UpdatePrimitives();
```

Comment:
We should call NeedsUpdate here instead of directly calling UpdatePrimitives.  This makes sure that UpdatePrimitive is called only once (other systems could be calling NeedsUpdate on the same frame).

Reply:
Actually NeedsUpdate is called a few lines down, this should be enough right?  I think we can simply remove the call to UpdatePrimitives.

Reply:
I think what is needed though is to call NeedsRedraw(). Then UpdatePrimitives() (which sets m_NeedsRedraw to true) is not needed here anymore.

Reply:
Done!